morph-all-view-graphql-files.js

```diff
@@ -126,6 +142,9 @@ ${
       isUsingDataConfiguration ? ', ...configuration' : ''
     } })
 ${isUsingDataTransform ? '  let data = useDataTransform(props, rdata)' : ''}
+${isUsingDataOnChange ? '  let onChange = useDataOnChange(props)' : ''}
+${isUsingDataOnSubmit ? '  let onSubmit = useDataOnSubmit(props)' : ''}
```

Comment:
would we want to pass data (or some of the other the hooks before the onSubmit and onChange return) to the hooks?

Reply:
I was thinking it is not necessary to pass the data as it will come as a parameter of the functions when invoked. do you think it may be useful in case the logic depends on the initial value of the data? e.g. based on the shape or type of an entity we could even return different onChange/onSubmit functions.
now that I think about it, it probably makes sense to pass the `data` to give developers more options if needed. do you agree we should pass the transformed `data` instead of the raw result received from the query?